build.sbt

```diff
@@ -9,6 +9,6 @@ resolvers ++= Seq(
 )
 
 // Chisel 3.4
-libraryDependencies += "edu.berkeley.cs" %% "chisel-iotesters" % "1.5.1"
-libraryDependencies += "edu.berkeley.cs" %% "chiseltest" % "0.3.1"
-
+addCompilerPlugin("edu.berkeley.cs" % "chisel3-plugin" % "3.4.3" cross CrossVersion.full)
```

Comment:
What is this compiler plugin good for?

Reply:
It has to do with signal naming and should be included since Chisel3 3.4. At least that's what I get from [here](https://www.chisel-lang.org/chisel3/docs/explanations/naming.html).